src/css/components/Button.scss

```diff
@@ -38,243 +36,96 @@ $Button-padding: 0 scaleGrid(2);
   height: $Button-height;
   line-height: $Button-line-height;
   margin: 0;
+  min-width: $Button-height;
   padding: $Button-padding;
   text-decoration: none;
   transition: all $duration-short ease-in-out;
   vertical-align: middle;
   white-space: nowrap;
 
   &:hover,
-  &:active,
   &:focus {
-    background-color: $Button-bg-hover;
-    border-color: $Button-border-color-hover;
-    color: $Button-color;
+    border-color: blue(5);
+    color: blue(5);
     outline: none;
   }
 
-  &.is-active {
-    background-color: $Button-bg-hover;
-    border-bottom-width: 1px;
-    border-top-width: $Button-border-bottom-width;
-    color: $Button-color;
+  &:active, &.is-active {
+    border-color: blue(7);
+    color: blue(7);
   }
 
   &.is-disabled,
   &:disabled {
     &,
     &:hover,
     &:active {
-      background-color: $Button-bg;
-      border-color: $Button-border-color;
+      background-color: gray(2);
+      border-color: gray(5);
       cursor: not-allowed;
-      color: $Button-color;
-      opacity: $Button-disabled-opacity;
+      color: gray(5);
     }
   }
 }
 
+$permutations: (
+  "primary" blue(6) blue(5) blue(7),
+  "blue" blue(6) blue(5) blue(7),
```

Comment:
Is this to accommodate our existing Button--blue and Button--primary classes... or for theming? Since these are the same color now, ideally we wouldn't need both, and could just keep Button--primary. I know marketing uses Button--blue, but I don't know if we want to keep around a duplicate class for that reason.

Reply:
this was to accommodate our existing classes. Happy to remove it as part of V4.